plugins/entry_replacement/lib/api/KalturaPluginOptionsArray.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * @package plugins
+ * @subpackage api
+ */
+class KalturaPluginOptionsArray extends KalturaTypedArray
```

Comment:
This name is too generic, if the array elements are named `KalturaPluginReplacementOptionsItem`, this class should be named KalturaPluginReplacementOptionsArray
@tan-tan-kanarek do you agree?